src/System.Net.Primitives/tests/FunctionalTests/Resources/System.Net.Primitives.Functional.Tests.rd.xml

```diff
@@ -0,0 +1,7 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<Directives xmlns="http://schemas.microsoft.com/netfx/2013/01/metadata">
+  <Library Name="System.Net.Primitives.Functional.Tests">
+    <Type Name="System.Net.CookieVariant" BinaryFormatter="All" />
+    <Type Name="System.Net.CookieCollection" BinaryFormatter="All" />
+  </Library>
+</Directives>
```

Comment:
That's a good explanation.  I think we should summarize that and add it as comments in the rd.xml file for these tests.  Otherwise, we won't remember why we added it.